MachOKit/Shared/MKPointer+Node.m

```diff
@@ -58,7 +64,13 @@ - (instancetype)initWithOffset:(mk_vm_offset_t)offset fromParent:(MKBackedNode*)
         };
     }
     
-    return [self initWithOffset:offset fromParent:parent context:context error:error];
+    return [self initWithOffset:offset fromParent:parent mask:mask context:context error:error];
+}
+
+//|++++++++++++++++++++++++++++++++++++|//
+- (instancetype)initWithOffset:(mk_vm_offset_t)offset fromParent:(MKBackedNode*)parent targetClass:(Class)targetClass error:(NSError**)error
+{
+    return [self initWithOffset:offset fromParent:parent mask:(mk_vm_address_t)(~0) targetClass:targetClass error:error];
```

Comment:
Please make the mask parameter here consistent with the one in `initWithOffset:fromParent:context:error:` by casting to `mk_vm_address_t` before the Binary NOT operation.

Reply:
Fixed in 5a86727